src/components/Todo/Todo.js

```diff
@@ -1,93 +1,126 @@
-import React, { useState, useEffect } from 'react';
-import Footer from '../Footer/Footer';
-import InputItem from '../InputItem/InputItem';
-import ItemList from '../ItemList/ItemList';
+import React, { useState } from "react";
+import Footer from "../Footer/Footer";
+import InputItem from "../InputItem/InputItem";
+import ItemList from "../ItemList/ItemList";
 import CardContent from "@material-ui/core/CardContent";
 import styles from "./Todo.module.css";
 
 const Todo = () => {
   const initialState = {
-    items: [
-      {
-        value: "Create new app",
-        isDone: true,
-        id: 1
-      },
-      {
-        value: "Get new job",
-        isDone: false,
-        id: 2
-      },
-      {
-        value: "Find new friends",
-        isDone: true,
-        id: 3
-      },
-      {
-        value: "Learn new language",
-        isDone: false,
-        id: 4
-      }
-    ],
-    count: 4,
-    error: false
+    item: JSON.parse(localStorage.getItem("item")) || [],
+    filter: "all",
+    count: JSON.parse(localStorage.getItem("count")) || 0,
+    toDo: "",
+    error: false,
+    errorMessage: "Something went wrong..."
   };
 
-  const [items, setItems] = useState(initialState.items);
+  const [item, setItem] = useState(initialState.item);
   const [count, setCount] = useState(initialState.count);
+  const [filter, setFilter] = useState(initialState.filter);
   const [error, setError] = useState(initialState.error);
+  const [errorMessage, setErrorMessage] = useState(initialState.errorMessage);
 
   const onClickDone = (id) => {
-    const newItemList = (items.map((item) => {
+    const newItemList = (item.map((item) => {
       const newItem = { ...item };
-      if (item.id === id) {
-        newItem.isDone = !item.isDone;
+      if (newItem.id === id) {
+        newItem.isDone = !newItem.isDone;
       }
       return newItem;
     })
     );
-    setItems(newItemList);
+    setItem(newItemList);
   };
 
   const onClickDelete = (id) => {
-    const newItems = items.filter((item) => item.id !== id);
-    setItems(newItems);
-    setCount((count) => count - 1);
+    const newItems = item.filter((item) => item.id !== id);
+    setItem(newItems);
+    setCount(count - 1);
   };
 
 
   const onClickAdd = (value) => {
-    if (value === "") {
+    const newDict = [];
+    for (let i in (item)){
```

Comment:
Codacy found an issue: [The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)